src/shell/command_line.cpp

```diff
@@ -374,3 +374,74 @@ void CommandLine::Shift(unsigned int amount)
 		}
 	}
 }
+
+bool CommandLine::FindBoolArgument(const std::string& name, bool remove,
+                                   char short_letter)
+{
+	const std::string double_dash = "--" + name;
+	const std::string dash        = '-' + name;
+	char short_name[3]            = {};
+	short_name[0]                 = '-';
+	short_name[1]                 = short_letter;
+	return FindExist(double_dash.c_str(), remove) ||
+	       FindExist(dash.c_str(), remove) ||
+	       (short_letter && FindExist(short_name, true));
```

Comment:
`FindExist(short_name, true)` — what is `true`? Suggest extracting it to a const with descriptive name.

Reply:
`true` was a bug in this case.  It was supposed to be the `remove` flag.  Fixed.

Reply:
Yeah in multi-arg functions it's almost never apparent what something like this means: `foo(blah, 4, 0, true, 233, false, false)` 🥶